.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: circleci/node:8.7
+      - image: circleci/node:12.4
```

Comment:
Should we switch to using node 12 in our dev setups as well? Also node 8 is getting deprecated soon.